README.md

```diff
@@ -1 +1,3 @@
-# hello-world
\ No newline at end of file
+# hello-world
+
+hello mt name is maria and i am  very tired 
```

Comment:
```suggestion
Hello my name is Maria and I am very tired,
```
Good to maintain proper grammar and spelling in codebase for easier to read documentation.

Reply:
+1